djconnectwise/tests/fixtures.py

```diff
@@ -1141,29 +1132,29 @@
 
 
 API_PROJECT_TICKET = {
-    'id': 47,
+    'id': 69,
     'summary': "Senior Engineer Project Planning",
     'isIssueFlag': False,
     'board': {
         'id': API_BOARD['id'],
         'name': API_BOARD['name'],
         '_info': {
-            'board_href': "https://connectwise.kerkhofftech.ca/v4_6_release/apis/3.0/service/boards/3"
+            'board_href': "https://example.com/v4_6_release/apis/3.0/service/boards/3"
         }
     },
     'status': {
         'id': 1,
         'name': "New",
         '_info': {
-            'status_href': "https://connectwise.kerkhofftech.ca/v4_6_release/apis/3.0/service/boards/3/statuses/1"
+            'status_href': "https://example.com/v4_6_release/apis/3.0/service/boards/3/statuses/1"
```

Comment:
When I said that your change from `example.com` to `connectwise.kerkhofftech.ca` shouldn't have been necessary, what I meant was that would have no effect on the tests, so it wasn't something you needed to do. I didn't mean go back and change any occurrence to `example.com` hahaha. 

To be clear, don't go back and change it back again. We will just leave it as is, as it has no effect. 

It's probably better anyway for a different reason, we don't want references to `connectwise.kerkhofftech.ca` in this repo. I might make an issue to audit the repo for any other references to KTI.